nx-truncate-ellipsis RSC-115

glenhunter opened the pull request:
https://issues.sonatype.org/browse/RSC-115

Added an example page for the `.nx-truncation-ellipsis` mixin. 

As part of this I added the `.nx-` prefix to the mixin to make it consistent with our other styles. I then refactored existing instances of the mixin in the RSC.

rpokorny commented:
> As part of this I added the .nx- prefix to the mixin to make it consistent with our other styles. I then refactored existing instances of the mixin in the RSC.

None of our other mixins except for `nx-font-family` start with `nx`.  Should they?  In particular the container mixins, which are already documented in the gallery?

glenhunter commented:
> > As part of this I added the .nx- prefix to the mixin to make it consistent with our other styles. I then refactored existing instances of the mixin in the RSC.
> 
> None of our other mixins except for `nx-font-family` start with `nx`. Should they? In particular the container mixins, which are already documented in the gallery?

Probably. I changed this one mostly because I was *constantly* typing `nx-truncate-ellipsis`.
